certbot/util.py

```diff
@@ -186,8 +186,8 @@ def make_or_verify_dir(directory, mode=0o755, uid=0, strict=False):
         if exception.errno == errno.EEXIST:
             if strict and not check_permissions(directory, mode, uid):
                 raise errors.Error(
-                    "%s exists, but it should be owned by user %d with"
-                    "permissions %s" % (directory, uid, oct(mode)))
+                    "%s exists, but it should be owned by user %s with"
+                    "permissions %s" % (directory, str(uid), oct(mode)))
```

Comment:
Do we need to make this change? Testing locally on Windows 10, tests pass locally without this. Also, problems here likely indicate `uid` is of an unexpected type and I'd rather have us fix that problem than just casting it to a string.

Reply:
Yes, it as been correctly more globally on compat.os_getuid().